pkg/scheduler/util/utils_test.go

```diff
@@ -122,55 +124,104 @@ func TestMoreImportantPod(t *testing.T) {
 	}
 }
 
-func TestRemoveNominatedNodeName(t *testing.T) {
+func TestClearNominatedNodeName(t *testing.T) {
+	t.Parallel()
+	statusErr := &apierrors.StatusError{
+		ErrStatus: metav1.Status{Reason: metav1.StatusReasonUnknown},
+	}
+
 	tests := []struct {
-		name                     string
-		currentNominatedNodeName string
-		newNominatedNodeName     string
-		expectedPatchRequests    int
-		expectedPatchData        string
+		name                  string
+		pods                  []*v1.Pod
+		patchError            error
+		expectedPatchErrors   []error
+		expectedPatchRequests int
+		expectedPatchData     string
```

Comment:
change type from string to []string so that we can compare the patch data easily.